app/modules/repo_manager/repo_manager.py

```
@@ -0,0 +1,479 @@
+        local_path = entry.get("local_path")
+        self._delete_metadata_entry(repo_name, branch, commit_id)
+
+        if local_path and os.path.exists(local_path):
+            try:
+                if os.path.isdir(local_path):
+                    shutil.rmtree(local_path)
+                else:
+                    os.remove(local_path)
+                logger.info("Deleted local repo copy at %s", local_path)
+            except OSError as exc:
+                logger.error("Failed to delete local repo copy %s: %s", local_path, exc)
```

Ruff (0.14.4)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)